src/components/Footer/FooterSecondary/StyledFooterSecondary.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components'
+import { resolveMedia } from '../../../utils/helpers'
+import { Row, Col } from '../../Grid'
+
+const FooterSecondaryContainer = styled(Row)`
+  ${props =>
+    !props.background && `background-color: ${props.theme.colors.mapsBlue};`}
+  ${props => !props.padding && `padding: 20px 0;`}
+  color: white;
+`
+
+const Acessibility = styled(Col)`
+  flex-grow: 0;
+  font-size: 0.875rem;
+  line-height: 1.5rem;
+`
+
+const MapsLink = styled(Col)`
+  flex-grow: 0;
+`
+
+const ListContainer = styled(Col)`
+  flex-grow: 0;
+  ${resolveMedia.md`
+    ${props => !props.margin && `margin-left: auto;`}
+  `}
+`
+
+const FooterSecondaryList = styled.ul`
+  display: flex;
+  flex-wrap: inherit;
+`
+
+const FooterSecondaryListItem = styled.li``
+
+export {
+  FooterSecondaryContainer,
+  Acessibility,
+  MapsLink,
+  ListContainer,
+  FooterSecondaryList,
+  FooterSecondaryListItem,
```

Comment:
I think we should also have a link to change the language in this section. This only appears in smaller viewports.